package.json

```diff
@@ -22,7 +22,7 @@
     "build": "wvr build --clean"
   },
   "dependencies": {
-    "@wvr/core": "2.2.5-rc.2",
+    "@wvr/core": "https://github.com/TAMULib/Weaver-UI-Core.git#piggy-back-poorly-scoped-requests",
```

Comment:
Could you revert this change?
I don't think we should commit github URLs to staging, etc...

Reply:
That version of `@wvr/core` is unrelated to the optimization for the steps your provided. I will gladly remove and wait further review there.